notebooks/115-async-api/README.md

```diff
@@ -0,0 +1,13 @@
+# Runtime Inference Optimization with Async API
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F115-async-api%2F115-async-api.ipynb)
+
+
+This notebook demostrate how to use [asynchronous execution](https://docs.openvino.ai/nightly/openvino_docs_deployment_optimization_guide_common.html) to improve data pipelining. 
+
+OpenVINO™ Runtime supports inference in either synchronous or asynchronous mode. Using the Async API can improve application’s overall frame-rate, because rather than wait for inference to complete, the app can keep working on the host, while the accelerator is busy. 
```

Comment:
"the app can keep working on the host, while the accelerator is busy. " Could you elaborate a bit more on what does the host mean and what the accelerator stands for here?

Reply:
Maybe could add the explanation of how the sync mode and the async mode works (such as the figures included in the Jupyter notebook) in the Readme as well, to help the readers to learn what the difference is when entering this notebook folder.

Reply:
Thanks for your time to review, the content is updated accordingly